manifests/o/​OpenStreetMap/OpenStreetMap.Josm.yaml

```diff
@@ -0,0 +1,23 @@
+PackageIdentifier: OpenStreetMap.Josm
+Publisher: OpenStreetMap
+PackageName: JOSM
+PackageVersion: 17833
+License: CC-BY-SA and LGPL since April 2014
+Moniker: josm
+Tags:
+- map
+ShortDescription: OpenStreetMap.Josm
+Description: JOSM is an extensible editor for ​OpenStreetMap (OSM) for ​Java 8+.
+PackageUrl: https://josm.openstreetmap.de/
+Installers:
+- Architecture: x86
+  InstallerUrl: https://josm.openstreetmap.de/download/windows/josm-setup.exe
```

Comment:
Link to specific version? https://josm.openstreetmap.de/download/windows/josm-setup-17833.exe